src/Bridge/Symfony/Resources/config/doctrine_orm.xml

```diff
@@ -20,7 +20,7 @@
         </service>
 
         <service id="fidry_alice_data_fixtures.persistence.purger.doctrine.orm_purger"
```

Comment:
I have to create two additional services for the ODMs as well, but i think this service can be left as is, as `orm` is present in its name.
The addtional services should be `fidry_alice_data_fixtures.persistence.purger.doctrine_mongodb.purger` and `fidry_alice_data_fixtures.persistence.purger.doctrine_phpcr.purger`.

Is it ok for you?